src/static/js/modules/gallery.js

```diff
@@ -0,0 +1,42 @@
+function imageGallery() {
+    const ACTIVE_CLASS = "active";
+    const carouselSlide = document.querySelectorAll(".item-slide");
+    const previews = document.querySelectorAll(".gallery-img");
+    const reset = document.querySelector(".gallery-wrapper-miniture img")
+
+    previews.forEach((preview) => {
+        preview.addEventListener("click", function () {
+            let size = carouselSlide[0].clientWidth;
+            let counter = this.id;
+            previews.forEach((preview) => preview.classList.remove(ACTIVE_CLASS));
+            preview.classList.add(ACTIVE_CLASS);
+            carouselSlide.forEach((cl) => {
+                cl.style.transform = "translateX(" + -size * counter + "px)";
+            });
+
+        });
+        preview.addEventListener("mouseover", function () {
+            const size = carouselSlide[0].clientWidth;
+            let counter = this.id;
+            previews.forEach((preview) => preview.classList.remove(ACTIVE_CLASS));
+            preview.classList.add(ACTIVE_CLASS);
+            carouselSlide.forEach((cl) => {
+                cl.style.transform = "translateX(" + -size * counter + "px)";
+            });
+
+        });
```

Comment:
All these functionality can be removed and you can have the same effect if you use `:hover` selector in your CSS.

Reply:
No, they cannot be removed since with `:hover` I cannot change what the active class is and I cannot calculate how much they should be moved and to which side they should be moved.

Reply:
Gallery images should not change when hovering, only when the image is clicked. What should happen when hovering is that the blue border should appear.

Reply:
alright, I had it like that because i thought it looked good but i will change it